packages/ts/react-crud/src/header-filter.tsx

```diff
@@ -179,7 +172,15 @@ export function DateHeaderFilter(): ReactElement {
       <DatePicker
         value={filterValue}
         placeholder="Filter..."
-        i18n={datePickerI18n}
+        i18n={{
```

Comment:
Could also be memoized, assuming the formatter is memoized too.

Reply:
I decided that memoizing formatter using `useMemo` does not worth it since it is just a simple instance of the class with couple of objects within and does not do any complex calculations. So it is just created from the provided locale each time.